### PR TITLE
Notice board

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR

This PR adds a notice board from the Paradise Station for the chef and maybe cargo.

## Why / Balance

I believe it will easier ordering and remembering the food orders. 

## Media

image_2023-11-03_13-30-36

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: Tunguso4ka
- add: CentCom allocated funds for the purchase of notice boards for kitchens

### CODE REVIEW

**Comment by UbaserB** [comment]
i like it

**Comment by AjexRose** [comment]
more space for more papers

File: Resources/Prototypes/Entities/Structures/Wallmounts/noticeboard.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            min: 1
+            max: 2
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
+  - type: Storage
+    maxSlots: 6
+    maxItemSize: Small
+    whitelist:
+      tags:
+        - Folder
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Structures/Wallmounts/noticeboard.yml:45]: folder? why?

Comment (CatMagic1) [reply Resources/Prototypes/Entities/Structures/Wallmounts/noticeboard.yml:45]: > folder? why?

so you can only put paper on it

**Comment by Tunguso4ka** [comment]
> more space for more papers

done

**Review by Emisse — APPROVED** [review]

**Pull request merged**